### PR TITLE
Suggested new templates

### PR DESCRIPTION
**Pull request by ronnno**
New suggested templates.

Suggestion - perhaps we can use labels instead of the title initials?

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE/breaking.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (electricmonk) [inline PULL_REQUEST_TEMPLATE/breaking.md:1]: maybe add link to spec issue/PR?

**Review by electricmonk — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE/breaking.md (hunk 1 above)
Comment (ronnno) [reply PULL_REQUEST_TEMPLATE/breaking.md]: it is implied in line 9:
`Keep it short... Any resolved issues?`

Comment (ronnno) [reply PULL_REQUEST_TEMPLATE/breaking.md:1]: it is implied in line 9:
`Keep it short... Any resolved issues?`

**Comment by ronnno** [comment]
@bolshchikov can we start experimenting with these templates in orbs-network-go?

**Pull request merged**